.gitignore

```diff
@@ -59,4 +59,5 @@ env[23]/
 
 # Project specific ignores
 static/js/modules/*.js
-static/js/dist/*.js
\ No newline at end of file
+static/js/dist/*.js
+cache.tmp/
```

Comment:
Please add a newline, otherwise the `git ignore` command [from git-extras](https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore) will fail.